docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -0,0 +1,200 @@
+local g = import 'grafana-builder/grafana.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'node-cluster-rsrc-use.json':
+      local legendLink = '%s/dashboard/file/k8s-node-rsrc-use.json' % $._config.grafana_prefix;
+
+      g.dashboard('USE Method / Cluster')
+      .addRow(
+        g.row('CPU')
+        .addPanel(
+          g.panel('CPU Utilisation') +
+          g.queryPanel(|||
+            (
+              instance:node_cpu_utilisation:avg_rate1m
+            *
+              instance:node_num_cpu:sum
+            / ignoring (instance) group_left
+              sum without (instance) (instance:node_num_cpu:sum)
+            )
+          |||, '{{instance}}', legendLink) +
+          g.stack +
+          { yaxes: g.yaxes({ format: 'percentunit', max: 1 }) },
+        )
+        .addPanel(
+          // TODO: Is this a useful panel?
```

Comment:
No, there's more than the numbers of CPUs involved in load

Reply:
I tend to agree, but this is again something I would like to discuss separately, as the original author certainly had thoughts and opinions here, too, so that I don't just want to kick this panel out.